src/pages/home/_trade-types.js

```diff
@@ -34,7 +34,7 @@ const TradeTypes = () => {
                     </Header>
                     <Text size="2.4rem" mt="1.6rem" mb="5rem">
                         {localize(
-                            'Trade what you like, the way you like it, and on your preferred market.',
+                            'Explore different trade types to trade on your preferred market.',
```

Comment:
please make a different PR for this change, and based to crowdin

Reply:
Ok will revert changes here and separate to another PR